docs/sql/string.md

```diff
@@ -186,6 +186,14 @@ addition to the normal way of `''`.
         If delimiter is absent assume it is the string <code>','</code>.</td>
     <td><code>SPLIT('a|b|c|', '|')</code> => { 'a', 'b', 'c', '' }</td>
   </tr>
+  <tr>
+    <td><code>SPLIT_PART(string, delimiter, n)</code></td>
+    <td>Extracts the <code>n</code>'th part of the string, splitting it at each occurrence of the delimiter.
```

Comment:
I would emphasize that this is a 1-based indexing scheme.

Reply:
Addressed in latest PR, thank you for review!